e2e-tests/cypress/integration/cookies-consent-bar.feature

```diff
@@ -1,36 +1,17 @@
-@wip
-Feature: Cookie consent bar
+Feature: Cookie consent bar - no JS
 
-  As a performance analyst working on the appeals service
-  I want users to be able to say Yes to GA cookies
-  So that I will receive analytics data for the service
+  As a PO on the appeals service
+  I need the cookie banner to cater for users who don’t have JS enabled
+  So that the service is compliant with the GDS standards
 
-  @as-98 @as-98-1a
-  Scenario: Cookie banner available until actioned - no decision
-    Given a user has not previously submitted cookie preferences
-    When the user neither accepts nor rejects not necessary cookies
+  @as-98 @as-98-1
+  Scenario: Cookie banner always displays
+    Given a user visits the site with JavaScript disabled
+    When the user navigates through the service
     Then the cookie banner remains visible
 
-  @as-98 @as-98-1b
-  Scenario: Cookie banner available until actioned - accepted
-    Given a user has not previously submitted cookie preferences
-    When the user accepts not necessary cookies
-    Then the accepted cookie banner becomes visible
-
-  @as-98 @as-98-1c
-  Scenario: Cookie banner available until actioned - rejected
-    Given a user has not previously submitted cookie preferences
-    When the user rejects not necessary cookies
-    Then the rejected cookie banner becomes visible
-
-  @as-98 @as-98-2
-  Scenario: Accept not necessary cookies
-    Given a user has not previously submitted cookie preferences
-    When the user accepts not necessary cookies
-    Then the GA cookies are enabled
-
-  @as-98 @as-98-3
-  Scenario: Reject Not necessary Cookies
-    Given a user has not previously submitted cookie preferences
-    When the user rejects not necessary cookies
-    Then the GA cookies remain disabled
+#  @as-98 @as-98-2
```

Comment:
@rcmsoftwaredevelopment please uncomment this one and ensure it passes as part of your current ticket - should need no work, but ping me if it does